lib/linguist/languages.yml

```diff
@@ -1279,6 +1279,7 @@ E:
   color: "#ccce35"
   extensions:
   - ".E"
+  - ".e"
```

Comment:
Whoops. I missed this yesterday. Linguist is not case-sensitive so we don't need this.

Reply:
Okay, I removed `".E"` from that list then.